docs/deployment.rst

```diff
@@ -1,63 +1,93 @@
 Deployment
 ==========
 
-moz-dev-dash is designed with `12-factor app philosophy`_ to run on `heroku`_, so you
-can easily deploy your changes to your own heroku app with `heroku toolbelt`_.
+moz-dev-dash is designed with `12-factor app philosophy`_, so you can easily
+deploy your changes to your own app.
 
 
-Deploy your own
----------------
+Deploy your own (on Mozilla Deis)
+---------------------------------
 
-#. `Create a heroku remote`_. We suggest naming it moz-dev-dash-`username`::
+#. `Install the deis client`_.
 
-    heroku apps:create moz-dev-dash-username
+#. `Register`_/login with the Mozilla Deis controller::
 
-#. Set the heroku app to use the "multi" buildpack::
+    deis register http://deis.deis.dev.mozaws.net
+    deis login
 
-    heroku buildpacks:set https://github.com/ddollar/heroku-buildpack-multi.git
+#. Add your public key::
 
-#. Push code to the heroku remote::
+    deis keys:add
 
-    git push heroku master
+#. Create the application::
 
-#. `Migrate`_ DB tables on heroku::
+    deis create dev-dashboard-username
 
-    heroku run python manage.py migrate
+#. Push code to the deis remote::
 
-#. Create a superuser on heroku::
+    git push deis master
 
-    heroku run python manage.py createsuperuser
+#. `Request a dev IAM account from Mozilla Cloud Ops`_ to create your RDS
+   Postgres instance in the Mozilla account.
 
-#. Open the new heroku app::
+#. Create an RDS Postgres instance in us-east-1 with default settings except:
```

Comment:
Is there more information on how to do this for a beginner? Does it require signing up [here](https://aws.amazon.com/rds/postgresql/)?